src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolutionResult.cs

```diff
@@ -1043,7 +1043,7 @@ private bool HadConstructedParameterFailedConstraintCheck(
             // is, then just report that error.
 
             MethodSymbol method = (MethodSymbol)(Symbol)result.Member;
-            if (!method.CheckConstraints(new ConstraintsHelper.CheckConstraintsArgs(compilation, conversions, includeNullability: false, location, diagnostics)))
+            if (!method.CheckConstraintsIncludingExtension(new ConstraintsHelper.CheckConstraintsArgs(compilation, conversions, includeNullability: false, location, diagnostics)))
```

Comment:
>CheckConstraintsIncludingExtension [](http://example.com/codeflow?start=24&length=34)

Is this change covered by a test?

Reply:
We don't report detailed overload resolution diagnostics for extensions, so I'll undo this.